### PR TITLE
Feat/client/activity generator

### LINKED ISSUE #10: feat/client: activity generator

### PR DESCRIPTION
**Pull request by Lartifex**
resolve #10

### CODE REVIEW

File: client/.env.example [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ORGANIZATION='the-organization-you-belong-to-or-default-organization'
```
Comment (jianingroja) [inline client/.env.example:1]: what is this referred to?

File: client/src/components/GenerateForm/GenerateForm.tsx [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+interface IFormInput {
+  Topic: {};
+  KidsNumber: {};
+  AgeRange: {};
+  Difficulty: {};
+  Place: {};
+  Duration: {};
+}
+
+const Form: React.FC = () => {
```
Comment (jianingroja) [inline client/src/components/GenerateForm/GenerateForm.tsx:14]: the component's name can be the same as file name, to keep it consistent

File: client/src/components/GenerateForm/GenerateForm.tsx [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        render={({ field }) => (
+          <Select {...field} options={placeOptions} placeholder={'Place...'} />
+        )}
+      />
+      <br />
+      <Controller
+        name="Duration"
+        control={control}
+        render={({ field }) => (
+          <Select
```
Comment (jianingroja) [inline client/src/components/GenerateForm/GenerateForm.tsx:135]: this piece of code is repeating itself many times, maybe there can be a function that takes the inputs and return a `<Select />` accordingly.

**Review by jianingroja — COMMENTED** [review]
the others lgtm, we can discuss how to implement it for now, or save it for later refactoring when necessary

File: client/.env.example (hunk 1 above)
Comment (Lartifex) [reply client/.env.example:1]: OpenAI needs this info from your personal profile to manage api

**Review by jianingroja — APPROVED** [review]

**Pull request merged**